### PR TITLE
GSOC2024

### PR DESCRIPTION
**Pull request by fcerullo-cycubix**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2024ideas.md [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+Tips to get you started in no particular order:
+- Read the
+  [Student Guidelines](gsoc2024).
+- Check our
+  [GitHub organization](https://github.com/OWASP).
+- Contact one of the project mentors below.
+
+## List of Project Ideas
+
+### [OWASP ZAP](https://zaproxy.org/)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2024ideas.md:46]: 😢 you should remove the whole ZAP section

File: pages/initiatives/gsoc/gsoc2024ideas.md [hunk 2]
```diff
@@ -0,0 +1,791 @@
+---
+layout: full-width
+title: GSoC 2024 Ideas
+tags: gsoc
+permalink: /initiatives/gsoc/gsoc2024ideas
+---
+
+# {{page.title}}
+
+[ZAP](#owaspzap) &bull; [Bug Logging Tool (BLT)](#bug-logging-tool-blt) &bull; [Maryam](#owaspmaryam) &bull; [SecureTea](#owasp-securetea) &bull; [PyGoat](#owasp-pygoat) &bull; [RiskAssessmentFramework](#owasp-risk-assessment-framework) &bull; [Juice Shop](#owaspjuiceshop) &bull; [OWASP WrongSecrets](#owasp-wrongsecrets) &bull; [OWASP DevSecOps Maturity Model](#owaspdevsecops-maturity-model) &bull; [OWASP secureCodeBox](#owasp-securecodebox) &bull; [OWASP ModSecurity Core Rule Set](#owasp-modsecurity-core-rule-set) &bull; [OWASP Nettacker](#owasp-nettacker)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2024ideas.md:10]: I guess you should drop ZAP here too.

```diff
- [ZAP](#owaspzap) &bull; 
```

**Comment by fcerullo-cycubix** [comment]
Ive just removed ZAP

**Review by kingthorin — APPROVED** [review]
LGTM

I haven't tested locally or checked against other years.

**Pull request merged**